AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -3156,7 +3155,7 @@ amp.usatoday.com,thewirecutter.com##.newsletter-container
 novmak.com##.newsletter-content
 diatribe.org##.newsletter-featured
 theblockcrypto.com##.newsletter-feed
-haberturk.com,mel.fm,washingtonpost.com,belsat.eu,economist.com,gitlab.com##.newsletter-form
+eff.org,guichevirtual.com.br,haberturk.com,mel.fm,washingtonpost.com,belsat.eu,economist.com,gitlab.com##.newsletter-form
```

Comment:
looks like this element is already hidden by 
eff.org##.pane-effector-signup

Reply:
Yeah, you're right. I will remove the domain. Site was in HTTPS Exclusions.